overlord/snapstate/handlers_components.go

```diff
@@ -496,8 +551,12 @@ func (m *SnapManager) doUnlinkComponent(t *state.Task, _ *tomb.Tomb) (err error)
 	if err != nil {
 		return err
 	}
-	cref := compSetup.CompSideInfo.Component
 
+	if err := saveCurrentKernelModuleComponents(t, snapSup, snapSt); err != nil {
```

Comment:
if I comment this out only linkCompSnapSuite.TestDoUnlinkComponent  itself fails, are we missing high level tests for this part of the functionality?

Reply:
Since we don’t yet support refreshing to a snap and losing some components (I have it as part of a future PR), this won’t be exercised yet. Want me to remove it from this PR and add it back later?

Reply:
thx, it's fine to keep, but please leave a TODO about testing that you can remove later